hypha/apply/projects/views/vendor.py

```diff
@@ -47,15 +48,13 @@ def dispatch(self, request, *args, **kwargs):
         project_settings = ProjectSettings.for_request(request)
         if not project_settings.vendor_setup_required:
             raise PermissionDenied
-        is_admin = request.user.is_apply_staff
         project = self.get_project()
-        is_owner = request.user == project.user
-        if not (is_owner or is_admin):
-            raise PermissionDenied
-        if not project.editable_by(request.user):
-            raise PermissionDenied
+        # is_owner = request.user == project.user   :todo: confirm it
```

Comment:
@frjo  Project owner(applicant) can edit vendor details? I think it was only for staff right?

Reply:
I think the applicant should be able to edit. @wes-otf can you confirm or deny this?

Reply:
hey sorry I'm so late here - I'll get confirmation on this monday along with testing it